# HG changeset patch
# User Rik <rik@octave.org>
# Date 1344011653 25200
#      Fri Aug 03 09:34:13 2012 -0700
# Node ID 8a62283e37839a0844889d5ed9c5b179e90e02a0
# Parent  60ff2cef569db8840bc1d021083502f9fc25ed57
maint: Add DO NOT EDIT messages to src/*.in.h for propagation to generated .h files.

* interpfcn/defaults.in.h, mxarray.in.h, oct-conf.in.h, version.in.h:
Add DO NOT EDIT messages for propagation to generated .h files.

diff --git a/src/interpfcn/defaults.in.h b/src/interpfcn/defaults.in.h
--- a/src/interpfcn/defaults.in.h
+++ b/src/interpfcn/defaults.in.h
@@ -1,9 +1,9 @@
-// defaults.h.in
+// DO NOT EDIT!  Generated automatically from defaults.in.h by configure
 /*
 
 Copyright (C) 1993-2012 John W. Eaton
 
 This file is part of Octave.
 
 Octave is free software; you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the
diff --git a/src/mxarray.in.h b/src/mxarray.in.h
--- a/src/mxarray.in.h
+++ b/src/mxarray.in.h
@@ -1,8 +1,9 @@
+// DO NOT EDIT!  Generated automatically from mxarray.in.h by configure
 /*
 
 Copyright (C) 2001-2012 Paul Kienzle
 
 This file is part of Octave.
 
 Octave is free software; you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the
diff --git a/src/oct-conf.in.h b/src/oct-conf.in.h
--- a/src/oct-conf.in.h
+++ b/src/oct-conf.in.h
@@ -1,9 +1,9 @@
-// oct-conf.h.in
+// DO NOT EDIT!  Generated automatically from oct-conf.in.h by configure
 /*
 
 Copyright (C) 1996-2012 John W. Eaton
 
 This file is part of Octave.
 
 Octave is free software; you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the
diff --git a/src/version.in.h b/src/version.in.h
--- a/src/version.in.h
+++ b/src/version.in.h
@@ -1,8 +1,9 @@
+// DO NOT EDIT!  Generated automatically from version.in.h by configure
 /*
 
 Copyright (C) 1992-2012 John W. Eaton
 
 This file is part of Octave.
 
 Octave is free software; you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by the
